### PR TITLE
improvement(perf-tests): add possibility to run perf tests on K8S

### PR DESCRIPTION
**Pull request by vponomaryov**
Also add 'latency' and 'throughput' pipelines for EKS backend.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-scylla-enterprise.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!groovy
```
Comment (roydahan) [inline jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-scylla-enterprise.jenkinsfile:1]: I would have separate this commit to a different PR to ease the merging of this.

Comment (vponomaryov) [reply jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-scylla-enterprise.jenkinsfile:1]: It is in separate commit

File: jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-scylla-enterprise.jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+longevityPipeline(
+    backend: 'k8s-eks',
+    aws_region: 'eu-north-1',
+    test_name: 'longevity_test.LongevityTest.test_custom_time',
+    test_config: 'test-cases/scylla-operator/longevity-scylla-operator-3h.yaml',
+    email_recipients: 'qa@scylladb.com,scylla-operator@scylladb.com',
+    post_behavior_db_nodes: 'destroy',
+    post_behavior_loader_nodes: 'destroy',
+    post_behavior_monitor_nodes: 'destroy',
+    timeout: [time: 400, unit: 'MINUTES']
```
Comment (bentsi) [inline jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-scylla-enterprise.jenkinsfile:14]: remove

**Review by bentsi — APPROVED** [review]

**Comment by roydahan** [comment]
It can't continue running from opeartor-1.2 branch.
I suggest to backport it to branch-perf-v11 and set perf jobs to be run from this branch.
You'll need to run it cross versions for long time.

**Comment by vponomaryov** [comment]
> It can't continue running from opeartor-1.2 branch.
> I suggest to backport it to branch-perf-v11 and set perf jobs to be run from this branch.
> You'll need to run it cross versions for long time.

The Idea behind dedicated branches is to keep code more stable. 
And no one is going to test operator/k8s compatibility for changes made in non-operator-related branches such as the one you propose to use for operator testing.

**Comment by roydahan** [comment]
The performance jobs are different from the regular jobs.
They need to run from a fixed branch that doesn't change  anything in the
tests in order to make the results comparable.
